torchci/components/metrics/panels/TablePanel.tsx

```diff
@@ -95,6 +102,7 @@ export function TablePanelWithData({
       density={"compact"}
       rows={data}
       columns={columns}
+      hideFooter={!showFooter}
```

Comment:
This part was removed in #4745 and this adds the parameter back but makes it configurable.